nilearn/datasets.py

```diff
@@ -449,6 +445,8 @@ def _fetch_file(url, data_dir, resume=True, overwrite=False,
     # Determine filename using URL
     parse = _urllib.parse.urlparse(url)
     file_name = os.path.basename(parse.path)
+    if file_name == '':
```

Comment:
Actually we already know that the variable is a string (see call to `basename` above). But I must admit that I don't like to write `if file_name:`. Reminds me of C code.